approach.md

```diff
@@ -0,0 +1,60 @@
+# OSEP proposed approach
+
+a) Document scope of analysis using STPA
+
+*In collaboration with other WGs e.g. Automotive?*
+
+* Assumed system context
+    - Hardware features, role of OS, boundaries of analysis, etc
+    - Doesn't have to be concrete / complex
+      - Specific to topic: start simple and elaborate later!
+* Losses
+    - OS-related outcomes that *may* violate a system's safety goals
+      - i.e. lead to harm in a safety-related system
+* Hazards
+    - OS-level system conditions that *may* lead to these losses
+* System-level constraints
+    - Criteria that must be satisfied to *prevent* or *mitigate* hazards
+      - May be a simple inversion of the hazard
+
+b) Identify and document system measures and mitigations for Linux
+
+*Out of scope for OSEP - LFSCS group?*
+
+* Provided by kernel features, compiler, hardware, etc
+* Find supporting evidence (design, tests, processes) for these
+* Identify responsibilities of components (kernel, compiler, etc)
+
+c) Perform hazard and risk analysis using STPA
+
+ *In collaboration with LFSCS and Safety Architecture WGs?*
+
+* Based on inputs from (a) and (b)
+* Document control structure(s) for topic
+    - Identify controllers and responsibilities
+      - Functional abstraction of system elements
+      - e.g. Kernel or subsystems, complier, other tools, etc
+    - Define interactions as control actions and feedback
+    - Identify Unsafe Control Actions and Loss Scenarios
```

Comment:
from Gab: I guess unsafe control actions are those that are associated to hazards, hence leading to loss scenarios, correct?

Reply:
Yes, UCAs are control actions that, for a particular set of system conditions, *can* lead to one or more hazards. Loss scenarios typically describe the causal factors that lead to UCAs, but may also describe other scenarios (not associated with a Control Action) that can result in hazards.